distribution/docs/src/main/resources/_contents/_configuring/configuring-web-service-providers-contents.adoc

```diff
@@ -0,0 +1,7 @@
+
+==== Configuring ${branding} Web Service Providers
+
+By default Solr, STS server, STS client and the rest of the services use the system property `org.codice.${ddf-branding}.system.hostname` which is defaulted to 'localhost' and not to the fully qualified domain name of the ${branding} instance.
```

Comment:
Pretty sure the system property name always uses `ddf` and isn't based on the branding.